### PR TITLE
EventHub: Added schema for 2018-01-01-preview

### PR DESCRIPTION
**Pull request by v-Ajnava**

### CODE REVIEW

**Comment by c4rlosmarin** [comment]
Hi there,

Just checking if we have an ETA for this?

Thanks!

**Comment by v-Ajnava** [comment]
@Tiano2017 can you please review the PR?

File: schemas/2015-01-01/deploymentTemplate.json [hunk 1]
```diff
@@ -671,6 +671,10 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.EventHub.json#/resourceDefinitions/namespaces_eventhubs_authorizationRules" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.EventHub.json#/resourceDefinitions/namespaces_eventhubs_consumergroups" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.EventHub.json#/resourceDefinitions/namespaces_disasterRecoveryConfigs" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-01-01-preview/Microsoft.EventHub.json#/resourceDefinitions/namespaces" },
```
Comment (TianoMS) [inline schemas/2015-01-01/deploymentTemplate.json]: >namespaces [](start = 137, length = 10)

we have a new root schema: 2019-04-01/deploymentTemplate.json. Can you add them to that root schema too?

Comment (v-Ajnava) [reply schemas/2015-01-01/deploymentTemplate.json]: Resolved

**Pull request merged**